.npmrc

```diff
@@ -0,0 +1 @@
+@sap:registry=http://nexus.wdf.sap.corp:8081/nexus/content/groups/build.releases.npm/
```

Comment:
This is an internal URL, please use `npm.sap.com` instead!

Reply:
yes, will be replaced once the SDK release is out.